lib/test_queue/runner.rb

```diff
@@ -361,6 +372,46 @@ def distribute_queue
       end
     end
 
+    def check_quorum(remote_workers)
+      return unless quorum_failed?(remote_workers)
+      quorum_failed(remote_workers)
```

Comment:
I don't love the `quorum_failed?` vs. `quorum_failed`. Any suggestions for better names?